examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -59,21 +96,41 @@ def test_attention_mask(self, utils):
 
 
 class TestVideoEncoder:
-    @pytest.fixture
-    def start(self):
+    @pytest.fixture(autouse=True)
+    def set_seed(self):
         set_rng_seed(1234)
 
+    @pytest.fixture
+    def utils(self):
         def make_input_video(c_dim=1):
             input_shape = [2, 3, 32, 32, 32]
-            input_shape[c_dim] = 3
-            return torch.randint(10, input_shape).float()
+            input_video = torch.randint(10, input_shape).float()
+            input_video = (
+                shift_dim(input_video, 1, c_dim) if c_dim != 1 else input_video
+            )
+            return input_video
 
         return make_input_video
 
-    def test_forward(self, start):
-        make_input_video = start
+    def test_forward_pretrained(self, utils, mocker):
+        make_input_video = utils
+        input_video = make_input_video()
+        patch_load_model(mocker)
+        encoder = VideoEncoder(pretrain_path=get_asset_path("S3D_sample.pt"))
```

Comment:
Just curious, how do you make the `S3D_sample.pt` and how can the size is very small (only 93kb)

Reply:
I randomly selected some percentage (i think 2% but i forgot) of keys from the actual pretrained state dict and discarded the other 98%